### PR TITLE
Rewrote readme's, added easy installer, cleaned up code

### PR DESCRIPTION
**Pull request by Pyenb**
Hey, 
I tought that maybe this could be nice. 
I rewrote most of the readme files and python code to be more structured. I also included an Innosetup file so that the programm can easily be installed with an installer, instead of copying files. It also adds it to the auto start menu and can the uninstalled from the windows settings.

Let me know if you dont' like anything.

### CODE REVIEW

File: Steel_battery_status.py
Comment (yurtemre7) [inline Steel_battery_status.py:1]: I would like to keep the mouse.py as the name of the main script.

File: innosetup/setup.iss
Comment (yurtemre7) [inline innosetup/setup.iss:1]: Add the EXE files in the `.gitignore` file.

File: innosetup/SMBR_setup.exe
Comment (yurtemre7) [inline innosetup/SMBR_setup.exe:1]: Add the EXE files in the .gitignore file.

File: readme.md [hunk 1]
```diff
@@ -1,82 +1,91 @@
...
+4. Uninstall Python 3 if you no longer need it.
 
-### Thanks to
+## Acknowledgements
 
-- [DeveloperX19](https://github.com/DeveloperX19) for the license of his
-  intellectual property of the art of the icon.
-- [flozz](https://github.com/flozz) for the `rivalcfg` library and the idea of a standalone python executable. 
+- [DeveloperX19](https://github.com/DeveloperX19) for the license of his intellectual property of the icon art.
+- [flozz](https://github.com/flozz) for the `rivalcfg` library and the idea of a standalone Python executable.
```
Comment (yurtemre7) [inline readme.md:92]: include yourself here please :)

**Comment by Pyenb** [comment]
Should be all resolved now

File: readme.md [hunk 2]
```diff
@@ -1,82 +1,92 @@
...
-     (C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup).
-   - You may need to change the bat script to point to your python installation
-     if you have multiple versions installed or renamed the executable.
-   - or you can run the script manually: `python3 mouse.py`
+1. Download the Git repository as a zip file and extract it somewhere (or clone it with Git).
+2. Install Python 3 (<https://www.python.org/downloads/>) and ensure you check the box to add it to your PATH.
+3. Install the following packages with Pip:
+   - `pip install rivalcfg`
+   - `pip install pystray`
+   - `pip install pillow`
```
Comment (yurtemre7) [inline readme.md]: Please reset the old pip installs with the specific versions, as I only tested on them. x)

Comment (Pyenb) [reply readme.md]: resolved

**Pull request merged**